app/views/images/show.html.haml

```diff
@@ -0,0 +1,41 @@
+%div.container-fluid.col-sm-5.col-sm-offset-3
+  %div.instagram-container.container-fluid.col-sm-12
+    %div.container-fluid.col-sm-12#username
+      %div.container-fluid.col-sm-8
+        = @image.user.email
+      %div.container-fluid.col-sm-4
+        = relative_time(@image.created_at)
+    %div#imagecontainer.container-fluid.col-sm-12
+      %div.container-fluid.col-sm-10.col-sm-offset-1
+        - if @image.image_file_name
+          = image_tag @image.image.url(:medium), :id => "image"
+    %div#imagecontainer.container-fluid.col-sm-12
+      %div.container-fluid.col-sm-8
+        = link_to @image.name.capitalize, image_path(@image)
+      %div.container-fluid.col-sm-4
+        = link_to "", image_likes_path(@image), method: :post, :class => "glyphicon glyphicon-thumbs-up"
+        = @image.likes.length
+        = 'like'.pluralize @image.likes.length
+    - if current_user == @image.user
+      %div#imagecontainer.container-fluid.col-sm-12
+        %div.container-fluid.col-sm-8
+          = link_to "Edit #{@image.name}", edit_image_path(@image)
+        %div.container-fluid.col-sm-4
+          = link_to "Delete #{@image.name}", image_path(@image), method: :delete
+    - if current_user
+      %div.container-fluid.col-sm-7
+        = link_to "Add a comment", new_image_comment_path(@image)
+    %div.container-fluid.col-sm-7
+    - if @image.comments.any?
+      - @image.comments.reverse.each do |comment|
+        %div#imagecontainer.container-fluid.col-sm-12
+          %div.container-fluid.col-sm-8
+            = comment.user.email
+          %div.container-fluid.col-sm-4
+            = relative_time(comment.created_at)
+          %div.container-fluid.col-sm-8
```

Comment:
`%div.container-fluid.col-sm-8` can be written as `.container-fluid.col-sm-8` since `%div` is implicit